### PR TITLE
add style

### PR DESCRIPTION
**Pull request by boo-shehab**
# Hi Reviewer 👋 
 
in this PR I added the styling for the bookstore using Usman's project because I couldn't open the design on Zeplin and these are the Project requirements

### Project requirements

- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](./sneak_peek_v2_0.md) of the project).
- Open the project *Bookstore CMS*.
- Add any missing JSX elements to your projects (without adding extra functionality)
- You may use the CSS rules detailed in Zeplin to style the website.
- Your final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -31,12 +32,17 @@ const Form = () => {
...
+
+    <section>
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form className="add-form" onSubmit={newBook}>
+        <input type="text" name="title" className="input title-input" value={book.title} placeholder="Book title" required onChange={handleChange} />
+        <input type="text" name="title" className="input author-input" value={book.author} placeholder="Book author" required onChange={handleChange} />
+        <button className="primary-button-big" type="submit">ADD BOOK</button>
+      </form>
+    </section>
   );
```
Comment (julie-ify) [inline src/components/Form.js:46]: - [ ] **[OPTIONAL]** I am unable to a book, I would recommend you fix this issue

File: src/components/Book.js [hunk 2]
```diff
@@ -19,17 +21,33 @@ const Book = (props) => {
...
+              <button className="button-outline" type="button">Comments</button>
+              <div className="vertical-divider" />
+              <button className="button-outline" type="button" onClick={remove}>Remove</button>
+              <div className="vertical-divider" />
+              <button className="button-outline" type="button">Edit</button>
+            </div>
+          </div>
+        </div>
+      </div>
     </>
```
Comment (julie-ify) [inline src/components/Book.js]: - [ ] Kindly add the other missing styling on the right hand side of the book component as contained in the design.

  Your Book component | Design component from book
  :---:|:---:
  dydyd | urfyfhf

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @boo-shehab,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 2 above)
Comment (boo-shehab) [reply src/components/Book.js]: Hi @julie-ify 
Thank you for reviewing my project I know about the right side of the card but they didn't ask us to make it even in the API there is no data for the time or the progress I'm in module 3 block 3

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @boo-shehab👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**